packages/components/badge/src/Badge.stories.tsx

```diff
@@ -0,0 +1,73 @@
+import { Icon } from '@spark-ui/icon'
+import { LikeFill } from '@spark-ui/icons/dist/icons/LikeFill'
+import { Radio, RadioGroup } from '@spark-ui/radio'
+import { Meta, StoryFn } from '@storybook/react'
+import { type ComponentProps } from 'react'
+
+import { Badge } from '.'
+
+type BadgeProps = ComponentProps<typeof Badge>
+
+const meta: Meta<typeof Badge> = {
+  title: 'Components/Badge',
+  component: Badge,
+}
+const sizes: BadgeProps['size'][] = ['sm', 'md']
+const intents: BadgeProps['intent'][] = ['error', 'info']
+
+const fakeAvatar = <div className="w-sz-40 h-sz-40 bg-outline rounded-lg" />
+
+export const Default: StoryFn = _args => <Badge>{fakeAvatar}</Badge>
+export const Standalone: StoryFn = _args => <Badge aria-label="New notification" />
+export const StandaloneWithCount: StoryFn = _args => <Badge count={1} />
+export const StandaloneWithinAnotherElement: StoryFn = _args => (
+  <RadioGroup>
+    <Radio value="foo">
+      Foo <Badge size="sm" count={3} aria-label={({ count }) => `${count} notifications`} />
+    </Radio>
+  </RadioGroup>
+)
+export const WrappingAnIcon: StoryFn = _args => (
+  <Badge size="sm" count={3}>
+    {/** TODO: Check the real behaviour with icons in order to avoid custom styles on them. **/}
+    <Icon size="lg" className="mx-md">
```

Comment:
This class is adding unnecessary margins to the badge which is confusing. I think its better as:
```suggestion
    <Icon size="lg">
```